### ISSUE TITLE
Endpoint for starting/stopping rig(s)

### ISSUE THREAD

**Issue by lazytarget** [mine-your-business/myb-nicehash-api#1]
Hi!

I just found this project but I see that it is missing a method for starting or stopping rigs.

I believe this would be a great addition, not only for my use case :)

**Comment by piekstra** [comment]
Hi @LazyTarget ! 

Thanks for the suggestion! I can try to add that this week.

**Comment by piekstra** [comment]
@LazyTarget would you be able to checkout the branch associated with this PR: https://github.com/mine-your-business/myb-nicehash-api/pull/2 and give it a try? I don't have time to add tests currently but figure I can get the feature out there for you and published for others in the meantime.

Thanks!

**Comment by lazytarget** [comment]
Cool stuff! Works like a charm! 😄 

Used it like this:

```
from nicehash import NiceHashPrivateApi, NiceHashRigAction
api = NiceHashPrivateApi('https://api2.nicehash.com', '<ORG-ID>', '<KEY>', '<SECRET>', verbose=True)
api.get_mining_rigs()   # to find the rig-id or device-id

api.rig_action('<RIG-ID>', '', NiceHashRigAction.STOP)
api.rig_action('<RIG-ID>', '', NiceHashRigAction.START)
```

**Comment by piekstra** [comment]
Closing this issue as the requested feature is now available with the published `v1.1.0` :)
